firestore/testapp/Assets/Firebase/Sample/Firestore/UIHandlerAutomated.cs

```diff
@@ -600,7 +619,175 @@ public class UIHandlerAutomated : UIHandler {
         AssertEq(snap.ToDictionary(), null);
       });
     }
+    
+    Task TestReadDocumentFromMultiDB() {
+      return Async(() => {
+        // TODO(b/284877917): Remove the Firestore emulator check when multi-db is supported
+        // by production.
+        if (!IsFirestoreEmulatorAvailable()) {
```

Comment:
If you want, you could create function, so that you can write:

```
    Task TestReadDocumentFromMultiDB() {
        // TODO(b/284877917): Remove the Firestore emulator check when multi-db is supported by production.
        return OnlyRunAgainstEmulator(() => {
            // Create 2 firestore instances and populate with documents.
            FirebaseApp customApp = FirebaseApp.Create(FirebaseApp.DefaultInstance.Options, "Interact-multi-DB");
```